src/twisted/mail/imap4.py

```diff
@@ -1090,8 +1090,9 @@ def sendNegativeResponse(self, tag = None, message = b''):
         self._respond(b'NO', tag, message)
 
 
-    def sendUntaggedResponse(self, message, async=False):
-        if not async or (self.blocked is None):
+    def sendUntaggedResponse(self, message, _async=None, **kwargs):
+        _async = get_async_param(_async, **kwargs)
```

Comment:
shouldn't this trigger a deprecation warning?

I expect that in the future code should be updated to just use `_async`...

but since this is public API I am -1 for using `_async` as an argument name

and since this is used as a boolean flag, I am +1 for naming it like `isAsync` or some other name which implies that this is a boolean